[CARBONDATA-3791]Fix spelling, validating links for quick-start guide and configuration parameters (#3740)
apache/carbondata: docs/configuration-parameters.md

> | carbon.storelocation | spark.sql.warehouse.dir property value | Location where CarbonData will create the store, and write the data in its custom format. If not specified,the path defaults to spark.sql.warehouse.dir property. **NOTE:** Store location should be in HDFS or S3. |

Reviewer: This property is deprecated in 2.0, user should use only spark.sql.warehouse.dir.
a) I think we should remove it
b) But where do we mention about the work around for upgrade from 1.6 to 2.0
@kunal642 , @QiangCai : please check and give your suggestions for both these points

Author: I have not removed this property. I added in NOTE section about deprecated in carbondata 2.0 version.

Reviewer: remove it. They should not configure in 2.0 version

Author: done

| Properties                                | Description                                                  |
| ----------------------------------------- | ------------------------------------------------------------ |

> | carbon.options.bad.records.logger.enable  | To enable or disable bad record logger. CarbonData can identify the records that are not conformant to schema and isolate them as bad records. Enabling this configuration will make CarbonData to log such bad records.**NOTE:** If the input data contains many bad records, logging them will slow down the over all data loading throughput. The data load operation status would depend on the configuration in ***carbon.bad.records.action***. |                   |

Reviewer 2: ```suggestion
| carbon.options.bad.records.logger.enable  | To enable or disable a bad record logger. CarbonData can identify the records that are not conformant to schema and isolate them as bad records. Enabling this configuration will make CarbonData to log such bad records.**NOTE:** If the input data contains many bad records, logging them will slow down the overall data loading throughput. The data load operation status would depend on the configuration in ***carbon.bad.records.action***. |                   |
```

Author: done

> | carbon.options.bad.records.action         | This property can have four types of actions for bad records FORCE, REDIRECT, IGNORE and FAIL. If set to FORCE then it auto-corrects the data by storing the bad records as NULL. If set to REDIRECT then bad records are written to the raw CSV instead of being loaded. If set to IGNORE then bad records are neither loaded nor written to the raw CSV. If set to FAIL then data loading fails if any bad records are found. |

Reviewer 2: ```suggestion
| carbon.options.bad.records.action         | This property has four types of  bad record actions: FORCE, REDIRECT, IGNORE and FAIL. If set to FORCE then it auto-corrects the data by storing the bad records as NULL. If set to REDIRECT then bad records are written to the raw CSV instead of being loaded. If set to IGNORE then bad records are neither loaded nor written to the raw CSV. If set to FAIL then data loading fails if any bad records are found. |
```

Author: done

| carbon.options.is.empty.data.bad.record   | If false, then empty ("" or '' or ,,) data will not be considered as bad record and vice versa. |
| carbon.options.bad.record.path            | Specifies the HDFS path where bad records needs to be stored. |

> | carbon.custom.block.distribution          | Specifies whether to use the Spark or Carbon block distribution feature.**NOTE: **Refer to [Query Configuration](#query-configuration)#carbon.custom.block.distribution for more details on CarbonData scheduler. |

Reviewer 2: ```suggestion
| carbon.custom.block.distribution          | Specifies whether to use the Spark or Carbon block distribution feature. **NOTE: ** Refer to [Query Configuration](#query-configuration)#carbon.custom.block.distribution for more details on CarbonData scheduler. |
```

Author: done
